keps/sig-cli/0015-kubectl-plugins.md

```diff
@@ -0,0 +1,152 @@
+---
+kep-number: 15
+title: Kubectl Plugins 
+authors:
+  - "@juanvallejo"
+  - "@soltysh"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - TBD
+  - "@pwittrock"
+  - "@deads2k"
+  - "@liggitt"
+approvers:
+  - TBD
+  - "@deads2k"
```

Comment:
soltysh I would think.  We need to have more distributed power and he's a sig lead in the right sig